.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           sudo xcodebuild -downloadPlatform visionOS
           sudo xcodebuild -runFirstLaunch
       - name: Lint Podspec
-        run: bundle exec pod lib lint --verbose --fail-fast --swift-version=6.0
+        run: bundle exec pod lib lint --verbose --fail-fast --swift-version=6.0 --allow-warnings # Cocoapods v1.6 now warns about potential naming colisions. We can fix this in the next breaking change.
```

Comment:
The warning:
```
    - WARN  | xcodebuild:  /Users/runner/work/CacheAdvance/CacheAdvance/Sources/CacheAdvance/CacheAdvance.swift:22:20: warning: public generic class 'CacheAdvance.CacheAdvance<T>' shadows module 'CacheAdvance', which may cause failures when importing 'CacheAdvance' or its clients in some configurations; please rename either the generic class 'CacheAdvance.CacheAdvance<T>' or the module 'CacheAdvance', or see https://github.com/apple/swift/issues/56573 for workarounds
```

I've certainly been bitten by this before. Maybe we'll rename the `CacheAdvance` type in a future release. Or maybe we'll allow warnings forever.